core/src/main/scala/ai/h2o/sparkling/backend/utils/ProxyStarter.scala

```diff
@@ -92,8 +92,8 @@ object ProxyStarter extends Logging {
     val holder = handler.addServletWithMapping(classOf[H2OFlowProxyServlet], "/*")
 
     val ipPort = RestApiUtils.getLeaderNode(conf).ipPort()
-    holder.setInitParameter("proxyTo", s"${conf.getScheme()}://$ipPort${conf.contextPath.getOrElse("")}")
-    holder.setInitParameter("prefix", conf.contextPath.getOrElse("/"))
+    holder.setInitParameter("proxyTo", s"${conf.getScheme()}://$ipPort")
```

Comment:
this can be simplified to the variant on the right. Basically we proxy everything after the port